03_launch_mgmt_cluster.sh

```diff
@@ -55,11 +55,10 @@ CAPM3_RUN_LOCAL="${CAPM3_RUN_LOCAL:-false}"
 
 if [ "${EPHEMERAL_CLUSTER}" == "kind" ]; then
   IRONIC_HOST="${PROVISIONING_URL_HOST}"
-  BMO_CONFIG="ironic-outside-config"
   export IRONIC_HOST_IP="${PROVISIONING_IP}"
 else
   IRONIC_HOST="${CLUSTER_URL_HOST}"
-  BMO_CONFIG="ironic-keepalived-config"
+  BMO_CONFIG="$BMOPATH/ironic-deployment/keepalived"
```

Comment:
Check how this is used. This will not work as it is set now.

Reply:
Fixed